matchms/filtering/peak_processing/remove_profiled_spectra.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import numpy as np
+from matchms.Spectrum import Spectrum
+
+
+logger = logging.getLogger("matchms")
+
+
+def remove_profiled_spectra(spectrum: Spectrum, mz_window=0.5):
+    """Remove profiled spectra
+
+    Spectra are removed if within the mz_window of 0.5 of the highest peak at least 2 peaks next to the main peak are of
+    intensity > max_intensity/2.
+
+    Reproduced from MZmine.
+    https://github.com/mzmine/mzmine3/blob/master/src/main/java/io/github/mzmine/util/scans/ScanUtils.java#L609
+    """
+    peaks_n = spectrum.mz.shape[0]
+    if peaks_n < 5:
+        return spectrum
```

Comment:
```suggestion
    if peaks_n < 3:
        return spectrum
```

Reply:
I get that this is mostly to speed up the process.
But since the `5` here is a magic number coming without explanation it should be consistent with the later tested criterium.

Reply:
I agree. Won't speed it up that much more anyway.